### PR TITLE
change application short answer questions for nwHacks 2023

### PR DESCRIPTION
**Pull request by michelleykim**
## Description

- https://www.notion.so/nwplus/nwHacks-Application-Questions-4c1cb50af8764a178311a349ce053b5b
- change short answer 2 to "Describe a project (does not need to be a technical project) that you worked on and a useful skill that you learned from it. (Max 200 words)"
- add short answer 3, "What character (from a movie, show, book, etc.) do you relate to most and why? (Max 50 words)"

## Other considerations

- canvas part of short answer 3 is handled in other PR #500

### CODE REVIEW

File: src/components/ApplicationForm/Skills.js [hunk 1]
```diff
@@ -275,6 +268,27 @@ export default ({ refs, errors, formInputs, onChange, role, handleResume }) => {
...
         </FormGroup>
+        <FormGroup>
+          <QuestionHeading>question 21</QuestionHeading>
+          <SubHeading size="1.25em">
+            What character (from a movie, show, book, etc.) do you relate to most and why? (max 50
+            words)
+            <Required />
+          </SubHeading>
+          <StyledTextArea
+            maxWords="200"
```
Comment (meleongg) [inline src/components/ApplicationForm/Skills.js:279]: I think this needs to be updated to 50 words too

Comment (michelleykim) [reply src/components/ApplicationForm/Skills.js:279]: lol u right

**Comment by michelleykim** [comment]
i've talked to the wang brothers and they said we can add the questions first then add the canvas part (asap)

**Review by meleongg — COMMENTED** [review]
lgtm!!!

**Review by meleongg — APPROVED** [review]
lgtm

**Pull request merged**